frontend/src/metabase/visualizations/visualizations/PivotTable/PivotTable.styled.tsx

```diff
@@ -159,6 +162,26 @@ export const PivotTableRoot = styled.div<PivotTableRootProps>`
           border-top: 1px solid var(--mb-color-border) (props);
         `
       : null}
+
+  ${props =>
+    props.isEditing
+      ? css`
+          & {
+            user-select: none;
+          }
+
+          &::-webkit-scrollbar,
+          & *::-webkit-scrollbar {
+            display: none;
```

Comment:
Setting `overflow: hidden` to the table body grid on editing results in it being broken after the edit